keyboards/dumbo/rules.mk

```diff
@@ -0,0 +1,36 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```

Comment:
```suggestion
```

Reply:
I think it makes sense to keep this as a reference especially as the keyboard is designed with the intention of being used with the nice!nano which uses DFU instead of caterina.

Reply:
The docs are meant to serve as that reference, which is why we've removed it from the template.

Reply:
Allright, that's a fair point. I will remove it then :)